.github/workflows/sc-docker-standalone-bldr.yml

```diff
@@ -0,0 +1,61 @@
+name: sc-docker-standalone-bldr
```

Comment:
why a copy of existing workflow file has been created? please extend existing `sc-standalone-deb10.yml` and `sc-standalone-deb11.yml`

Reply:
Note, `sc-docker-standalone-bldr.yml` was renamed when the support of Deb11 environment is added.

Reply:
this has been resolved